### PR TITLE
Update companies.yml to add Levi Strauss & Co.

### PR DESCRIPTION
**Pull request by ppamidimarri**
Adds or updates the following companies, events, or universities:
 - 
 - 

#### Company updates
 - [x] I have put the most recent relevant date in the "Last Update" column
 - [ ] I have linked to the article about the change, not the company's homepage (Please put N/A if there is no public post)

#### Event updates
 - [ ] I have linked to the article about the change, not the event's homepage

#### University updates
 - [ ] I have linked to the article about the change, not the university's homepage

### CODE REVIEW

**Comment by ppamidimarri** [comment]
Added Levi Strauss & Co. information

File: _data/companies.yml [hunk 1]
```diff
@@ -1533,12 +1533,12 @@
   events: Restricted
   last_update: 2020-03-13
 
-- name: Lightstep
+- name: Levi Strauss & Co.
```
Comment (drewbrew) [inline _data/companies.yml]: It looks like you're trying to overwrite Lightstep here, which I don't think you want. Would you mind fixing this?

**Comment by krishnatre-siddhartha** [comment]
#1342 @ppamidimarri works as you wanted?

File: _data/companies.yml (hunk 1 above)
Comment (krishnatre-siddhartha) [reply _data/companies.yml]: I think its resolved with #1342

**Comment by drewbrew** [comment]
Closing in favor of #1342

**Pull request closed without merging**